tests/driver_lm75/Makefile

```diff
@@ -0,0 +1,10 @@
+include ../Makefile.tests_common
+
+DRIVER ?= tmp1075
+
+USEMODULE += $(DRIVER)
+USEMODULE += xtimer
+
+CFLAGS += -DTHREAD_STACKSIZE_MAIN="(3*THREAD_STACKSIZE_LARGE+FLASHPAGE_SIZE)"
```

Comment:
Why do we need such a large stack size?
And what has flashpage have to do with this?